### PR TITLE
Remove queue position from Dev Hub

### LINKED ISSUE #17451: Remove queue position from Dev Hub
The Dev Hub displays an extension's position in the review queue. Since we don't always review the queue by positional order, we should remove this display from the Dev Hub so we don't set false expectations around estimated wait time.

### PR DESCRIPTION
**Pull request by bhushan-borole**
Fixes #17451

### CODE REVIEW

**Comment by bhushan-borole** [comment]
Hey @diox 
I will have to remove these tests `test_in_queue` and `test_static_themes_in_queue` right?

**Comment by diox** [comment]
Yes

**Review by diox — CHANGES_REQUESTED** [review]
Nice work. Could you please also remove the `get_position()` function entirely ? I don't think it's used anymore.

**Comment by bhushan-borole** [comment]
Sure I will, No problem.

File: src/olympia/devhub/views.py [hunk 1]
```diff
@@ -126,9 +125,7 @@ def index(request):
         recent_addons = request.user.addons.all().order_by('-modified')[:3]
         ctx['recent_addons'] = []
         for addon in recent_addons:
-            ctx['recent_addons'].append(
-                {'addon': addon, 'position': get_position(addon)}
-            )
+            ctx['recent_addons'].append({'addon': addon})
```
Comment (diox) [inline src/olympia/devhub/views.py:128]: If you're up for going a little further, I just noticed this is now useless... 

The whole loop could be completely removed, and `recent_addons` assigned to `ctx['recent_addons']` directly. Then, in the template (`src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html`), you would modify the loop over `recent_addons` to do `{% for addon in recent_addons %}`, removing the `{% set addon = item.addon %}` bit.

Does that make sense ?

Comment (bhushan-borole) [reply src/olympia/devhub/views.py:128]: Oh okay, yes I got it.
Will make the changes.

**Comment by bhushan-borole** [comment]
@diox Can you tell me the command to run all the tests?

**Comment by diox** [comment]
If you have docker etc set up locally, you can use `make test` to run the whole suite, but it takes a while (and there might be a few unrelated failures locally). You can also do something like `make test ARGS=something` to pass arguments down. That can be used to restrict which file/directory containing the tests to use for instance (and access any `pytest` command-line arguments)

**Comment by bhushan-borole** [comment]
I was actually thinking of running this command, I copied this from the CI pipeline checks
`pytest -m "es_tests and not needs_locales_compilation and not static_assets" -v src/olympia/`
I have installed docker, but everything is yet to be set.

**Comment by diox** [comment]
That's fairly close to what `make test` does, but it would take a while to run them all locally as we have a bunch of tests.

I just noticed passing `ARGS` seems broken. Instead you can just add `docker-compose exec web` to your command-line (I personally use a small shell script that does exactly this)

**Comment by diox** [comment]
Side note: that `-m es_tests` would run the elasticsearch tests only, which are unlikely to be affected here. I would rather do something like `docker-compose exec web pytest src/olympia/devhub` to run all devhub tests, for instance. Maybe adding the `-m not needs_locales_compilation` to avoid tests needing locales.

**Comment by bhushan-borole** [comment]
Oh okay, it seems all the tests have passed, so I think I don't have to go through that hassle.

**Comment by bhushan-borole** [comment]
> Side note: that `-m es_tests` would run the elasticsearch tests only, which are unlikely to be affected here. I would rather do something like `docker-compose exec web pytest src/olympia/devhub` to run all devhub tests, for instance. Maybe adding the `-m not needs_locales_compilation` to avoid tests needing locales.

Ohh I see, that's actually a nice idea!

**Review by diox — APPROVED** [review]
Nice work, thanks!

**Comment by caitmuenster** [comment]
Thank you so much, @bhushan-borole! 😍

**Comment by bhushan-borole** [comment]
@caitmuenster Most Welcome!!!

**Pull request merged**